module/src/components/calendarView/calendarView.component.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+
+import { use as useForm } from '../../hooks/form';
+import { Arrays } from '../../utils/arrays';
+import { ClassNames } from '../../utils/classNames';
+import { Dates } from '../../utils/dates';
+import { Maths } from '../../utils/maths';
+import { Button } from '../button';
+import { NativeSelectInput } from '../nativeSelectInput';
+import { getDayOfWeekHeadings, getDays, getMonths, getYears, ICalendarHighlight, ICalendarHighlightParsed, IDay } from './calendarView.utils';
```

Comment:
worth splitting some of the more complex types into calendarView.types.ts for these more complex components or is that gonna be overkill?

Reply:
Yeah that probably makes sense